presto-parser/src/main/java/io/prestosql/sql/tree/AddColumn.java

```diff
@@ -27,22 +27,26 @@
 {
     private final QualifiedName name;
     private final ColumnDefinition column;
+    private final boolean tableNameExists;
+    private final boolean columnNameNotExists;
```

Comment:
`columnNotExists` here and in other statement classes

In other words `Name` is not misleading and should not be used in these variable names

Reply:
fixed, thanks!